php/src/Connection.cpp

```diff
@@ -348,7 +348,7 @@ handleConnectionInfoFreeStorage(zend_object* object)
 bool
 IcePHP::connectionInit(void)
 {
-    // We register an interface and a class that implements the interface. This allows  applications to safely include
+    // We register an interface and a class that implements the interface. This allows applications to safely include
```

Comment:
This comment looks bogus. It was most likely referring to the Slice generated code for _local_ Slice Connection and ConnectionInfo, which no longer exists.

Reply:
I removed the bogus comment.
Good catch.